kview/src/main/java/module-info.java

```diff
@@ -143,6 +143,11 @@
     opens dev.ikm.komet.kview.klwindows to javafx.fxml, org.carlfx.cognitive;
     opens dev.ikm.komet.kview.mvvm.viewmodel to dev.ikm.komet.application, javafx.fxml, org.carlfx.cognitive;
 
+    exports dev.ikm.komet.kview.mvvm.view.confirmation;
+    opens dev.ikm.komet.kview.mvvm.view.confirmation;
```

Comment:
Can we remove line 147?
Opens keyword is allowing any class to use reflection. This should not be open to the everyone.